python/tests/integration/test_client_server.py

```diff
@@ -111,27 +112,63 @@ def test_tchannel_call_request(tcp_server, call_response):
     hostport = 'localhost:%d' % (tcp_server.port)
 
     response = yield tchannel.request(hostport).send(endpoint, None, None)
+    assert response.args[0] == call_response.args[0]
+    assert response.args[2] == call_response.args[2]
 
-    assert response.arg_1 == call_response.arg_1
-    assert response.arg_3 == call_response.arg_3
+
+@pytest.fixture
+def load_data_from_file():
+    # create large args
+    data = ""
+    with open(__file__, 'r+') as file:
+        for _ in range(50):
+            data += file.read()
+            file.seek(0)
+
+
+# TODO test case will fail due to StreamClosedError when
+# increase the LARGE_AMOUNT to even bigger
+@pytest.mark.gen_test
+@pytest.mark.parametrize('arg2, arg3', [
+    ("", big_arg()),
+    (big_arg(), ""),
+    ("test", big_arg()),
```

Comment:
Ah, given how you're using `big_arg`, leave it as a function but move it to a separate util module.

`conftest` is for configuration. Try not to import it directly.